backend/backends/base.py

```
@@ -114,15 +148,26 @@ def get_torch_device(
         try:
             import torch_directml
 
-            if torch_directml.device_count() > 0:
-                return torch_directml.device(0)
+            device_count = torch_directml.device_count()
+            if device_count > 0:
+                device = torch_directml.device(0)
+                iris_detected = _detect_iris_igpu()
+                if iris_detected:
+                    logger.info(f"Using DirectML device (Intel Iris iGPU detected)")
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)